cmake/tpl/qhull.cmake

```diff
@@ -3,7 +3,11 @@ set(QHULL_URL "https://github.com/qhull/qhull/archive/2019.1.tar.gz")
 set(QHULL_CACHE "${CACHE_DIR}/2019.1.tar.gz")
 set(QHULL_SRC_DIR ${QHULL_PREFIX}/src/qhull/src)
 
-set(${lib_name}_libs libqhullstatic.a)
+if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+  set(${lib_name}_libs libqhullstatic_d.a)
+else()
+  set(${lib_name}_libs libqhullstatic.a)
+endif()
```

Comment:
I can't find anything in the README for qhull that mentions libqhullstatic_d.a, I only see libqhullstatic_r.a, but I'm assuming it exists when you need to build this in debug. The TPL scripts currently build all of our TPLs as release by default. Do you find you need to build local debug TPLs like this often?

Reply:
In Blast when we build debug, all the packages are built in debug, including qhull. When debugging, it makes things much easier, since you can dive into the third-party libraries. I can't remember where I say libqhullstatic_d, but if I run into it again I'll send you a link.